premerge/cluster-management.md

```diff
@@ -237,3 +237,70 @@ ensure they are in a state consistent with the terraform IaC definitions.
 
 [Strategies for Upgrading ARC](https://www.kenmuse.com/blog/strategies-for-upgrading-arc/)
 outlines how ARC should be upgraded and why.
+
+## Grafana tokens
+
+The cluster has multiple services communicating with Grafana Cloud:
+ - the metrics container
+ - per-node monitoring  (Grafana Alloy, Prometheus node exporter)
+ - per-cluster monitoring (Opencost, Alloy)
```

Comment:
Could you expand a bit on Opencast? Is it installed with Alloy?

Reply:
Added link to the helm documentation, but yes, this is part of the k8s-monitoring helm chart we use.